Sources/Purchasing/Purchases.swift

```diff
@@ -1631,6 +1636,33 @@ public extension Purchases {
 // MARK: Configuring Purchases
 public extension Purchases {
 
+    /**
+     * Configures an instance of the Purchases SDK with a specified ``Configuration``.
+     *
+     * The instance will be set as a singleton.
+     * You should access the singleton instance using ``Purchases/shared``
+     *
+     * - Note: Use this initializer if you wish to adjust network request timeouts.
+     *
+     * - Parameter configuration: The ``Configuration`` object you wish to use to configure `Purchases`
```

Comment:
I think we're missing a double-backtick at the end for \`\`Purchases\`\`

Reply:
I kinda liked it as single, since we're already in the file, but you're probably right- for consistency double back ticks makes sense.

Reply:
✅